src/movies.js

```diff
@@ -1,16 +1,79 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+    return directorsArray
+}
+
+const directorsArray = movies.map ((element) => {
```

Comment:
You needed to put that piece of code inside your functions and change which array you're targeting
```suggestion
const directorsArray = moviesArray.map ((element) => {
```